codespell_lib/data/dictionary.txt

```diff
@@ -27999,6 +28025,7 @@ shoudlnt->shouldn't
 shoudn't->shouldn't
 shoudn->shouldn
 shoudt->should, shouldn't,
+shoul->should
```

Comment:
```suggestion
shoul->should, shawl,
```

Reply:
Ah, you have to be a native speaker to find this one! Although _shawl_ also exists in French (spelled _châle_ nowadays), I would never have found it by myself.

Reply:
Probably quite unlikely, but I could possibly imagine someone typing it if they didn't know how to spell it!